docker/syncprojects.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# a recommended value for SYNC_INTERVAL is 30 --> 30 minutes
+
+if [ ! -z "$SYNC_INTERVAL" ]; then
+    syncInterval=$SYNC_INTERVAL
+    echo "syncInterval=$SYNC_INTERVAL"
+    echo "SYNC_INTERVAL=$SYNC_INTERVAL for syncing projects" >> /app/run_webApp.log
+    if((syncInterval > 5)); then
+        echo "doing startup delay of 60 seconds..."
+        sleep 60
+        echo "running loop"
+        while true
+        do
+           echo "calling sync_projects for user pontoon"
+           python /app/manage.py sync_projects > /app/lastSync.log
+           # multiply by 60 to have minutes in the env var
+           sleep $((syncInterval*60))
+        done
+    fi
+fi
+echo "SYNC_INTERVAL is not defined or <5, doing nothing."
+echo "SYNC_INTERVAL is not defined or <5, doing nothing." >> /app/run_webApp.log
```

Comment:
This looks like a very unreliable way to run sync. The first failure will break the whole process, and you'll have to restart the app manually to recover. Aren't there better solution for running jobs regularly out there, that don't require a fragile script like this?

Reply:
right, I tried to use a cron job, but with no success. For me this works for now. This is better than no sync at all. Feel free to improve. 
BTW. in k8s you will not try to restart the app or similar, you simply drop the entire pod.